pe.c

```diff
@@ -1074,18 +1075,23 @@ handle_image (void *data, unsigned int datasize,
 				       entry->vendor_version,
 				       entry->vendor_url);
 			}
+			res = parse_sbat_var(&sbat_var_entries);
+			if (res < 0)
+				console_print(L"could not parse SBAT variable");
```

Comment:
shouldn't this be an error instead of just an info log? And the error propagated ? This is another reason to make this function return an `EFI_STATUS`.